beet/library/resource_pack.py

```diff
@@ -350,10 +366,12 @@ class ResourcePack(Pack[ResourcePackNamespace]):
     # fmt: off
     blockstates:      NamespaceProxyDescriptor[Blockstate]     = NamespaceProxyDescriptor(Blockstate)
     models:           NamespaceProxyDescriptor[Model]          = NamespaceProxyDescriptor(Model)
+    equipment_Models: NamespaceProxyDescriptor[EquipmentModel] = NamespaceProxyDescriptor(EquipmentModel)
```

Comment:
This capital M should not exist

Reply:
We should probably call it `equipment` or `equipments` now? Since the folder moved in 1.21.4